go.mod

```diff
@@ -28,9 +28,9 @@ require (
 	go.opencensus.io v0.24.0
 	go.uber.org/automaxprocs v1.5.3
 	go.uber.org/zap v1.24.0
-	golang.org/x/net v0.23.0 //CVE-2023-45288; >=v0.23.0
+	golang.org/x/net v0.33.0 //CVE-2023-45288; >=v0.23.0
```

Comment:
We could remove this comment at this point I think

Reply:
True--I had forgotten I added those annotations. They're definitely less helpful too now that we're just doing `go get` instead of the replaces.